scripts/release.sh

```diff
@@ -36,6 +36,6 @@ function release {
 
 git submodule update
 
-for package in dhall-lsp-server dhall-openapi dhall-json dhall-yaml dhall-bash dhall-nix dhall-nixpkgs dhall-docs dhall; do
+for package in dhall-lsp-server dhall-openapi dhall-json dhall-yaml dhall-bash dhall-nix dhall-nixpkgs dhall-docs dhall-csv dhall; do
```

Comment:
Let's wait with this until this package is ready for an initial release.

```suggestion
for package in dhall-lsp-server dhall-openapi dhall-json dhall-yaml dhall-bash dhall-nix dhall-nixpkgs dhall-docs dhall; do
```

Reply:
Should I remove the references to dhall-csv from release.nix too or is it enough with removing it from scripts/release.sh?

Reply:
IIUC the `release.nix` file mostly defines what's built by Hydra. We want to include `dhall-csv` there, so we get CI for it.

Reply:
Ok thanks! I'll only change the release.sh then